build.gradle

```diff
@@ -194,12 +194,12 @@ dependencyUpdates.resolutionStrategy = {
         }
         rules.withModule("org.controlsfx:controlsfx") { ComponentSelection selection ->
             if (selection.candidate.version ==~ /9.*/) { // Reject version 9 or higher
-                 selection.reject("Cannot be updated to 9.*.* until Jabref works with Java 9")
+                selection.reject("Cannot be updated to 9.*.* until Jabref works with Java 9")
             }
         }
-		rules.withModule("com.github.tomtung:latex2unicode_2.12") { ComponentSelection selection ->
+        rules.withModule("com.github.tomtung:latex2unicode_2.12") { ComponentSelection selection ->
```

Comment:
I guess these formatting changes are copied from a different PR. Did you merged the latest changes from master?